ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageSingle.test.js

```diff
@@ -0,0 +1,175 @@
+    selectResourceFilterType,
+    typeAndEnterResourceFilterValue,
+    visitWorkloadCveOverview,
+} from './WorkloadCves.helpers';
+import { selectors } from './WorkloadCves.selectors';
+
+describe('Image Single page', () => {
+    withAuth();
+
+    it('should correctly handle Image single page specific behavior', () => {
+        visitWorkloadCveOverview();
+
+        // View the 'Image' tab on the overview page and click any image link
+        cy.get(selectors.entityTypeToggleItem('Image')).click();
+        cy.get('tbody tr td[data-label="Image"] a').first().click();
+
+        // Check that only applicable resource menu items are present in the toolbar
+        cy.get(selectors.resourceDropdown).click();
+        cy.get(selectors.resourceMenuItem('CVE'));
+        cy.get(selectors.resourceMenuItem('Image')).should('not.exist');
+        cy.get(selectors.resourceMenuItem('Deployment')).should('not.exist');
+        cy.get(selectors.resourceMenuItem('Cluster')).should('not.exist');
+        cy.get(selectors.resourceMenuItem('Namespace')).should('not.exist');
+        cy.get(selectors.resourceDropdown).click();
+
+        // Check that the CVEs by severity totals in the card match the number in the "results found" text
+        const cardSelector = selectors.summaryCard('CVEs by severity');
+        cy.get(
+            [
+                `${cardSelector} svg + *:contains("Critical")`,
+                `${cardSelector} svg + *:contains("Important")`,
+                `${cardSelector} svg + *:contains("Moderate")`,
+                `${cardSelector} svg + *:contains("Low")`,
+            ].join(',')
+        ).then(($severityTotals) => {
+            const severityTotal = $severityTotals
+                .toArray()
+                .reduce((acc, $el) => acc + parseInt($el.innerText.replace(/\D/g, ''), 10), 0);
+
+            const plural = severityTotal === 1 ? '' : 's';
+            cy.get(`*:contains(${severityTotal} result${plural} found)`);
+        });
+
+        // Check that the CVEs by status totals in the card match the number in the "results found" text
+        const fixStatusCardSelector = selectors.summaryCard('CVEs by status');
+        cy.get(
+            [
+                `${fixStatusCardSelector} svg + *:contains("with available fixes")`,
+                `${fixStatusCardSelector} svg + *:contains("without fixes")`,
+            ].join(',')
+        ).then(($statusTotals) => {
+            const statusTotal = $statusTotals
+                .toArray()
+                .reduce((acc, $el) => acc + parseInt($el.innerText.replace(/\D/g, ''), 10), 0);
+
+            const plural = statusTotal === 1 ? '' : 's';
+            cy.get(`*:contains(${statusTotal} result${plural} found)`);
+        });
+    });
+
+    it('should correctly apply severity filters', () => {
+        visitWorkloadCveOverview();
+
+        // Go to the image tab
+        cy.get(selectors.entityTypeToggleItem('Image')).click();
+
```

Comment:
Although pro and con of selectors in test file versus encapsulated in helpers file, comment and multiple calls is some evidence in favor of a semantically named helper function with `'Image'` as argument.